docs/Getting-Started-frontend.md

```diff
@@ -303,6 +304,97 @@ meta: {
 
 ----
 
+#### How does the Logical layer (Model) of the website works?
+
```

Comment:
@SebinSong I would like to know your feedback too in these new docs explaining how the Logical part of GI works.

Reply:
Hi @sandrina-p :) As to this topic, I can't help but admit that figuring out what sbp does is the one of the things I struggled(I am still, actually) with the most on this project. I will write a more detailed comment on this for you soon!

Reply:
Hope I haven't kept you waiting for too long @sandrina-p :). I just read through Information-flow.md you authored and I feel this kind of documentation will definitely be very helpful for people who are not familiar with 'Selector Based Programming', like me. Figuring out 'which selector' does 'what' just by going through the source codes hasn't been that successful for me so far. so I always wish there was some sort of a guide document that lists sbp selectors being used in the app and tells what they are responsible for and when to use them.
I also love how you let the readers know which file to look at for a certain concept.

Reply:
Cool @SebinSong ! One last tip for you: When you need to look for what a selector does, search in the project for `'selector-name:` (with : at the end without `/create`). Most of the times, it will find where the selector is declared and what it does, instead of showing all the results where the selector is used!

Ex:
```
`'gi.contracts/group/proposalVote/create'` -> shows everywhere the selector is called
`'gi.contracts/group/proposalVote':` -> shows where the selector is declared.
```

Reply:
And to add to that: this is just for contract action selectors, two of which are dynamically created by `Contract.js` for each action. For all other selectors in the project, they are registered normally (with no dynamic creation like `\`${action}\`/process` (himm, not sure how to use markdown to create backticks in backticks...)

Reply:
@taoeffect @sandrina-p Thanks for your informative tips! :)